CADETProcess/modelBuilder/carouselBuilder.py

```diff
@@ -128,13 +128,10 @@ def add_inter_zone_connections(self, flow_sheet):
 
                 flow_sheet.add_connection(origin, destination)
 
-        flow_rates = self.flow_sheet.get_flow_rates()
         for zone in self.zones:
```

Comment:
I wonder if this is even necessary here. I think that output states are entirely handled by `Events`.